metricbeat/module/docker/helper.go

```diff
@@ -52,24 +51,15 @@ func ExtractContainerName(names []string) string {
 	return strings.Trim(output, "/")
 }
 
-func BuildLabelArray(labels map[string]string) []common.MapStr {
+func BuildLabelArray(labels map[string]string) common.MapStr {
```

Comment:
OK! Just did a commit right now with the name ConvertContainerLabels, will rename them again to DeDotLabels (as they are directly related to Labels), and also add the comments!